growbox/server.py

```diff
@@ -15,19 +15,23 @@
 from datetime import datetime
 
 # Third party imports
-from flask import Flask, render_template, make_response, request, Markup
+from flask import render_template, make_response, request, Markup
 import requests
 from requests.exceptions import HTTPError
 from crontab import CronTab
 
 # Local application imports
-import usb_camera
-import serial_port
-import sqlite
+from growbox import app
+from growbox import usb_camera
+from growbox import serial_port
+from growbox import sql
```

Comment:
Better:

```python
from growbox import app, usb_camera, serial_port, sql
```

Or even better, if a thing is referenced just one-two times, use the fully qualified path:

```python
growbox.app
```